src/main/java/com/jakewharton/byteunits/BinaryByteUnit.kt

```diff
@@ -0,0 +1,263 @@
+
+    override fun toBytes(count: Long) =
+      UnitUtil.multiply(count, PB / B, MAX / (PB / B))
+
+    override fun toKibibytes(count: Long) =
+      UnitUtil.multiply(count, PB / KB, MAX / (PB / KB))
+
+    override fun toMebibytes(count: Long) =
+      UnitUtil.multiply(count, PB / MB, MAX / (PB / MB))
+
+    override fun toGibibytes(count: Long) =
+      UnitUtil.multiply(count, PB / GB, MAX / (PB / GB))
+
+    override fun toTebibytes(count: Long) =
+      UnitUtil.multiply(count, PB / TB, MAX / (PB / TB))
+
+    override fun toPebibytes(count: Long) =
+      count
+  };
+
+  /**
+   * Converts the given size in the given unit to this unit. Conversions from finer to coarser
+   * granularities truncate, so lose precision. For example, converting from `999` bytes to
+   * kibibytes results in `0`. Conversions from coarser to finer granularities with arguments
+   * that would numerically overflow saturate to [Long.MIN_VALUE] if negative or
+   * [Long.MAX_VALUE] if positive.
+   *
+   *
+   * For example, to convert 10 kilobytes to bytes, use:
+   * `ByteUnit.KIBIBYTES.convert(10, ByteUnit.BYTES)`
+   *
+   * [sourceCount] the size in the given [sourceUnit].
+   * [sourceUnit] the unit of the [sourceCount] argument.
+   * @return the converted size in this unit, or [Long.MIN_VALUE] if conversion would
+   * negatively overflow, or [Long.MAX_VALUE] if it would positively overflow.
+   */
+  open fun convert(sourceCount: Long, sourceUnit: BinaryByteUnit): Long {
+    throw AbstractMethodError()
+  }
+
+  /**
+   * Equivalent to [KIBIBYTES.convert(count, this)][.convert].
+   * [count] the bit count
+   * @return the converted count, or [Long.MIN_VALUE] if conversion would negatively
+   * overflow, or [Long.MAX_VALUE] if it would positively overflow.
+   */
+  open fun toKibibytes(count: Long): Long {
+    throw AbstractMethodError()
+  }
+
+  /**
+   * Equivalent to [MEBIBYTES.convert(count, this)][.convert].
+   * [count] the bit count
+   * @return the converted count, or [Long.MIN_VALUE] if conversion would negatively
+   * overflow, or [Long.MAX_VALUE] if it would positively overflow.
+   */
+  open fun toMebibytes(count: Long): Long {
+    throw AbstractMethodError()
+  }
+
+  /**
+   * Equivalent to [GIBIBYTES.convert(count, this)][.convert].
+   * [count] the bit count
+   * @return the converted count, or [Long.MIN_VALUE] if conversion would negatively
+   * overflow, or [Long.MAX_VALUE] if it would positively overflow.
+   */
+  open fun toGibibytes(count: Long): Long {
+    throw AbstractMethodError()
+  }
+
+  /**
+   * Equivalent to [TEBIBYTES.convert(count, this)][.convert].
+   * [count] the bit count
+   * @return the converted count, or [Long.MIN_VALUE] if conversion would negatively
+   * overflow, or [Long.MAX_VALUE] if it would positively overflow.
+   */
+  open fun toTebibytes(count: Long): Long {
+    throw AbstractMethodError()
+  }
+
+  /**
+   * Equivalent to [PEBIBYTES.convert(count, this)][.convert].
```

Comment:
From what I remember, these leading dots are a result of the converter and are unnecessary or maybe even break the kdoc.

Reply:
What should this statement in Kotlin Documentation look like? Previously it was like this: https://github.com/JakeWharton/byteunits/blob/72cefe731bdbeea0668c1b21b7a0a28f40c48ec8/src/main/java/com/jakewharton/byteunits/BinaryByteUnit.java#L231

Reply:
https://github.com/JakeWharton/byteunits/pull/24